kok-api/src/main/java/com/kok/kokapi/room/adapter/in/dto/response/RoomDetailResponse.java

```diff
@@ -6,16 +6,14 @@ public record RoomDetailResponse(
         String id,
         String roomName,
         int capacity,
-        MemberResponse member,
-        String roomLinkUrl
+        MemberResponse member
 ) {
     public static RoomDetailResponse from(Room room) {
         return new RoomDetailResponse(
                 room.getId(),
                 room.getRoomName(),
                 room.getCapacity(),
-                MemberResponse.from(room.getMember()),
-                room.getRoomLinkUrl()
+                MemberResponse.from(room.getMember())
```

Comment:
`Room에 속한 Member이니 Leader겠지?`라는 유추의 과정이 필요하다고 느껴졌어요!
메세지를 주는 메서드를 분리하면 어떨까요?
``
```suggestion
                MemberResponse.from(room.getLeader())
```
[소소한 인사이트(?) 공유><](https://developerfarm.wordpress.com/2012/02/01/object_calisthenics_10/)

Reply:
Good!